basiclti/basiclti-blis/src/java/org/sakaiproject/lti13/LTI13Servlet.java

```diff
@@ -1488,18 +1506,26 @@ protected void handleNamesAndRoles(String signed_placement, HttpServletRequest r
 				*/
 				sakai_ext.put("sakai_role", sakaiRole);
 
+				// TODO: Should we only do this if requested via groups=true ??
```

Comment:
Do we need the "TODO" comment?

Reply:
Yes - The sakai extension always included the groups - now that there is an official way to include the groups, I wonder if we should delete the sakai_extension.  MY theory is that I will keep them as if for a bit, let the new serivce find its place, then deprecate the sakai_extension and then follow the spec 100% (will improve performance).  But for now I want to rember to come back and do this.